src/lightning/data/streaming/dataset.py

```diff
@@ -368,13 +366,17 @@ def _validate_state_dict(self) -> None:
             )
 
 
-def _try_create_cache_dir(input_dir: str, shard_rank: int = 0) -> Optional[str]:
+def _try_create_cache_dir(input_dir: str, shard_rank: Optional[int] = None) -> Optional[str]:
```

Comment:
But now the shard_rank argument is useless, we don't use it anywhere else

Reply:
removed.